plugins/mysql/mysql_size_all

```diff
@@ -111,6 +111,7 @@ foreach my $db (getDBList()) {
 
     $total_size = $datas+$indexes;
   }
+  $db =~ s/\./-/g;
```

Comment:
The dash is not an allowed character, too. Maybe it works, but by specification it is forbidden. It surely causes problems if used in `cdef` evaluation.

Instead you should do the following:
* add `use Munin::Plugin;` at the top of the script
* replace all illegal characters: `$db = clean_fieldname($db);`

(I hope, I guessed the purpose of your change correctly)

Reply:
Hi, Thanks for reply.
Didn't insert the dash but i will check and change the lines from the original developer too.

Reply:
Indeed I meant `hyphen` instead of `dash`. Thanks for fixing this issue!